easybuild/tools/github.py

```diff
@@ -383,17 +387,25 @@ def fetch_easyconfigs_from_pr(pr, path=None, github_user=None):
                 _log.info("Downloading %s from %s" % (fn, full_url))
                 download_file(fn, full_url, path=os.path.join(path, patched_file), forced=True)
         else:
-            apply_patch(diff_filepath, path, level=1)
+            apply_patch(diff_filepath, dl_path, level=1)
+
+    if dl_path != path:
+        for eb_file in glob.glob(os.path.join(dl_path, 'easybuild', 'easyconfigs', '*')):
+            os.symlink(eb_file, os.path.join(path, os.path.basename(eb_file)))
+        check_path = dl_path
+
+    else:
+        check_path = path
```

Comment:
`dl_path` and `check_path` are now exactly the same thing, and I don't like either name :p

how about changing the definition of `dl_path` to `final_path`, and just use that (no need to introduce another variable for exactly the same thing)